ipywidgets/widgets/widget.py

```diff
@@ -353,6 +366,17 @@ def _lock_property(self, **properties):
             self._property_lock = {}
 
     @contextmanager
+    def _disable_read_only_ness(self):
```

Comment:
you are the native speaker. I let you pick the one you prefer. `_allow_write`, `_chmod_777`, `_rw` :)